app/views/shared/_navbar.html.slim

```diff
@@ -0,0 +1,8 @@
+nav.navbar.navbar-toggleable-md.navbar-inverse.bg-primary
+  .container
+    button.navbar-toggler.navbar-toggler-right[type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation"]
+      span.navbar-toggler-icon
+    a.navbar-brand[href="/"] Personal Finance App
+    #navbarNavAltMarkup.collapse.navbar-collapse
```

Comment:
@sinpai [check](https://cssguidelin.es/#naming) css `class` and `id` naming convention

Reply:
Changed 'navbarNavAltMarkup' to 'navbar-responsive-button', it would be more understandable.